src/manifest/MsQuicEtw.man

```diff
@@ -1779,6 +1790,15 @@
               symbol="QuicApiWaitOperation"
               value="14"
               />
+          <event
+              keywords="ut:LowVolume"
+              level="win:Informational"
+              message="$(string.Etw.PerfCountersRundown)"
+              opcode="win:Info"
+              symbol="QuicPerfCountersRundown"
```

Comment:
Add the necessary changes to quicetw and etwlib for these (mostly just header changes).